### DISCUSSION TITLE
Grouping Color Styles

### DISCUSSION THREAD

**Discussion by iansampaio** [six7/figma-tokens#606] [Q&A] [answered]
I'm trying to preserve the same structure I had for my styles before, where I have a folder/group for a color eg: Indigo and inside it all of its shades from `indigo.100` to `indigo.900`. 

<img width="269" alt="image" src="https://user-images.githubusercontent.com/439176/163448493-a07c80b4-cce3-48d7-8de6-5cf3197cd7a0.png">

<img width="237" alt="image" src="https://user-images.githubusercontent.com/439176/163448552-39142349-2341-4a88-8de3-23f26805b4e0.png">

Not only my exported style does not carry its folder, now I have all the colors duplicated.

**Comment by six7** [comment]
Could you provide some more info? I dont yet understand what's the expected outcome here / what the input is.

**Comment by iansampaio** [comment]
I was afraid my explanation wasn't good, so my apologies. 

I'd like to have my exported styles colors grouped & collapsable like the image below:

<img width="242" alt="image" src="https://user-images.githubusercontent.com/439176/163514702-f531a45d-98d1-4963-a463-9da28a9bce7d.png"> 

_you can see the `Indigo` group expanded and `Blue` and `Green` are collapsed_ 

In my attempts the exported style colors won't go inside the folder:
<img width="240" alt="image" src="https://user-images.githubusercontent.com/439176/163515062-73c73bea-2865-448e-bfca-014292ea8cb6.png">
_Note there is no group expanded or collapsed._

Tokens are named like `Indigo.900` so I thought they would translate to `Indigo`/`900`. 

What am I doing wrong?

**Reply by six7** [reply]
Can you check in the plugin settings and make sure `ignore first part of token name for styles` is turned off? that should solve it! 🙏

**Comment by iansampaio** [comment, marked as answer]
>Can you check in the plugin settings and make sure ignore first part of token name for styles is turned off? that should solve it! 🙏

This is the answer.
For some reason I thought this would affect how it was displayed/grouped in Figma Tokens instead of styles
